bin/profile/migrate.js

```diff
@@ -0,0 +1,212 @@
+                                        : `migration would be skipped because a Node 8 webtask already exists.`));
+                            }
+                        }
+                        else {
+                            console.log(Chalk.blue(`${w}:`), Chalk.green(
+                                args.yes 
+                                    ? `migrating...`
+                                    : `webtask would be migrated...`));
+                        }
+
+                        if (performMigration) {
+                            return node4Migration.migrate({ 
+                                containerName: profile.container,
+                                webtaskName: w,
+                                token: profile.token,
+                                dryRun: !args.yes
+                            }, (e,b) => {
+                                if (!reportUrl && b && b.reportUrl) {
+                                    reportUrl = b.reportUrl;
+                                }
+                                var m = b && b.warnings || [];
+                                var warnings;
+                                if (Array.isArray(m)) {
+                                    m.forEach(w => warnings = warnings ? `${warnings}\n* ${w.warningType}: ${w.message}` : `* ${w.warningType}: ${w.message}`);
+                                }
+                                if (e) {
+                                    console.log(Chalk.red(`...Error: ${e.message}`));
+                                    if (warnings) console.log(Chalk.red(warnings));
+                                    return cb(null, false);
+                                }
+                                else if (warnings) {
+                                    console.log(Chalk.yellow(`...Succeeded with warnings:\n${warnings}`));
+                                    return cb(null, true);
+                                }
+                                else {
+                                    console.log(Chalk.green(`...Success`));
+                                    return cb(null, true);
+                                }
+                            });
+                        }
+                        else {
+                            return cb(null, false);
+                        }
+                    }
+                }, (e) => { // Async.eachSeries
+                    if (e) return cb(e);
+                    console.log();
+                    if (reportUrl) {
+                        console.log(`Migration report is available at: ${reportUrl}`);
+                        console.log();
+                    }
+                    if (!args.yes) {
+                        console.log(Chalk.yellow(`${Chalk.bold('NOTE')} This was a simulation, no changes were made. To perform actual migration to Node 8, rerun this command and specify the --yes switch.`));
```

Comment:
To perform *the* actual migration to Node 8

Reply:
ok